control_panel_api/views.py

```diff
@@ -130,20 +131,46 @@ def perform_destroy(self, instance):
 
         instance.aws_delete_role()
 
-    @detail_route(permission_classes=[IsSuperuser])
-    def customers(self, request, pk=None):
+
+class AppCustomersAPIView(GenericAPIView):
+    queryset = App.objects.all()
+    serializer_class = GroupMemberSerializer
```

Comment:
I would also consider to rename `Group`/`GroupMember` to avoid any possible confusion with Django's Groups. `Auth0GroupSerializer`/`Auth0GroupMemberSerializer` may be a bit long winded but less ambiguous.

Reply:
Oh yes I totally missed that, thanks.